src/main/environment/common_ci.properties

```diff
@@ -12,7 +12,9 @@ carestreamOrderCreateURL =@env.COMMON_API_BASE_URL@/carestream/createOrder
 #  Registration
 registrationUrl =@env.COMMON_API_BASE_URL@/beneficiary/create
 
-syncSearchByLocation =@env.COMMON_API_BASE_URL@/id/searchByDistrictId
+syncSearchByLocation =@env.COMMON_API_BASE_URL@/id/searchByVillageIdAndLastModifiedDate
+
+getBenCountToSync = @env.COMMON_API_BASE_URL@/id/countBenByVillageIdAndLastModifiedDate
 
```

Comment:
Here we can use @env.HWC_IDENTITY_API_BASE@

So
@env.HWC_IDENTITY_API_BASE@ = base/hwc-identity-0.0.1/
and @env.COMMON_API_BASE_URL@ = base/hwc-commonapi-v1.0/

Reply:
Will you please also edit 

getBenImageFromIdentity =@env.COMMON_API_BASE_URL@/id/benImageByBenRegID
to 
getBenImageFromIdentity =@env.HWC_IDENTITY_API_BASE@/id/benImageByBenRegID

Reply:
Done.